### PR TITLE
Add mDevCamp 2018 to the list

### PR DESCRIPTION
**Pull request by davidvavra**

### CODE REVIEW

File: _conferences/mdevcamp_2018.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: "mDevCamp"
+website: https://mdevcamp.eu/
+location: Prague, Czechia
+
+date_start: 2018-6-15
+date_end:   2018-6-15
+
+cfp_site: https://docs.google.com/forms/d/e/1FAIpQLSeweXK_uC5kam0FZUiNDfXCQXEaWCkuutF9d8ilVFlbkurZlA/viewform
```
Comment (JakeWharton) [inline _conferences/mdevcamp_2018.md]: This won't show up without start and end dates.

File: _conferences/mdevcamp_2018.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: "mDevCamp"
+website: https://mdevcamp.eu/
+location: Prague, Czechia
+
+date_start: 2018-6-15
+date_end:   2018-6-15
+
+cfp_start: 2018-1-8
```
Comment (vanniktech) [inline _conferences/mdevcamp_2018.md]: Should

Comment (vanniktech) [inline _conferences/mdevcamp_2018.md:8]: Shouldn't those be prefixed with a leading zero?

Comment (JakeWharton) [reply _conferences/mdevcamp_2018.md:8]: It probably doesn't matter. Depends on how strict of a validating parser Jekyll has and it appears (from the CI build) to be lenient.

**Pull request merged**